Xamarin.Forms.ControlGallery.Android/PerformanceTrackerRenderer.cs

```diff
@@ -0,0 +1,77 @@
+﻿using AView = Android.Views.View;
+using System;
+using System.ComponentModel;
+using Xamarin.Forms;
+using Xamarin.Forms.ControlGallery.Android;
+using Xamarin.Forms.Controls;
+using Xamarin.Forms.Platform.Android;
+using static Android.Views.ViewTreeObserver;
+using Android.Views;
+
+[assembly: ExportRenderer(typeof(PerformanceTracker), typeof(PerformanceTrackerRenderer))]
+
+namespace Xamarin.Forms.ControlGallery.Android
+{
+	public class PerformanceTrackerRenderer : ViewRenderer, IOnDrawListener
```

Comment:
This lets me know approximately when the view has finally finished rendering so I can stop the clock.